i18n/en.json

```diff
@@ -4517,6 +4524,8 @@
   "status_dropdown.set_ooo": "Out of Office",
   "status_dropdown.set_ooo.extra": "Automatic Replies are enabled",
   "string.id": "default message",
+  "success_modal.return_to_workspace": "Return to workspace",
+  "success_modal.subtitle": "Your final bill will be prorated. Your workspace now has Cloud Starter limits.",
```

Comment:
Can we make this `has Cloud Starter limits` -> `has {planName} limits`? It will be less translation churn when we also support downgrading from enterprise to professional.